src/evict/evict_lru.c

```diff
@@ -1149,6 +1186,14 @@ __evict_push_candidate(WT_SESSION_IMPL *session,
 	evict->btree = S2BT(session);
```

Comment:
Not shown here, but there is a comment without a closing paren at line 1170 in this function.